src/aiida/cmdline/utils/decorators.py

```diff
@@ -239,7 +239,7 @@ def do_circus_stuff():
     return wrapped(*args, **kwargs)
 
 
-def deprecated_command(message):
+def deprecated_command(message, version=None):
```

Comment:
```suggestion
def deprecated_command(message: str, version: str = '3'):
```
I would set a default for the version here. For deprecations of API code we also usually default to removing it in v3.

Reply:
done